client/index.html

```diff
@@ -27,9 +27,13 @@
 		</div>
 
 		<div id="main">
-			<div class="col-md-12">
-				<h1>Dashboard</h1>
-			</div>
+
+            <div id="chart1" style="width: 45%; height: 45vh; float: left;"><canvas id="velocity-time"></canvas></div>
```

Comment:
Rename id of the divs `chart1`, `chart2`, etc. to something that's a bit more verbose eg. `velocityTimeChartDiv`